components/collapse/style/index.ts

```diff
@@ -188,9 +192,9 @@ export const genBaseStyle: GenerateStyle<CollapseToken> = (token) => {
             paddingInlineStart: padding,
 
             [`> ${componentCls}-expand-icon`]: {
-              height: fontSizeLG * lineHeight,
+              height: fontHeight,
```

Comment:
```suggestion
              height: fontHeightLG,
```
这里正确应该是 fontSizeLG * lineHeightLG，看了一下文字行高也不对，需要改一下

Reply:
1